README.md

```diff
@@ -39,6 +39,25 @@
     - 50002: Shippable marketing ui
     - 50003: Shippable admin panel
 
+### Load balancers
+- The usual(and recommended) way of exposing Shippable service end points is
+  via loadbalancers. The LB's are not required to be exposed to the public
```

Comment:
Don't need an apostrophe.  It's plural, not saying that something belongs to a load balancer.